README.md

```diff
@@ -332,6 +332,16 @@ The Platforms supported by the Node.js project are generally divided into four d
 
 *Deprecated* Platforms consist of prior *Primary*, *Secondary* or *Experimental* Platforms that are no longer under active development and are no longer supported by active Collaborators. Only the TSC can decide to move a *Primary* platform to *Deprecated* status. Support for *Deprecated* Platforms may be removed entirely after the next *semver-major* version increase.
 
+### Build Stability
+
+The stability of the Node.js Build process and infrastructure is critical for those who rely on the ability to build Node.js from source as part of their deployment workflow. The ability to support Node.js on multiple platforms requires that Collaborators avoid making breaking changes that require new or updated compiler versions, new C/C++ language level features, etc.
```

Comment:
"fixed" could mean simply declaring that the particular broken platform configuration is no longer supported past a given version. All I'm saying is that such changes ought to be reviewed a bit more so that the consequences are clear and understood and we know what is going to break.